src/graphql/schema.js

```diff
@@ -239,11 +239,30 @@ type PaginatedBiggestWinner {
   items: [BiggestWinner]!
 }
 
+type Leaderboard {
```

Comment:
`LeaderboardUser`? Leaderboard is the entity of the entire board itself so need to differentiate individual items.